lib/ext.core.TemplateHandler.js

```diff
@@ -107,7 +107,14 @@ TemplateHandler.prototype.onTemplate = function ( token, frame, cb ) {
 	// in <infobox> extension
 	frame.templateArgInfo = frame.templateArgInfo || {};
 	if(tgt && tgt.target) {
-		frame.templateArgInfo[tgt.target] = this.getArgInfo(state);
+		// infobox template can be used multiple times in single article
```

Comment:
I don't really have any suggestions at this moment - without having to look into the code more - unfortunately I can't do it anytime soon (busy with upstream sync). If this code works (and was tested really well) I would go forward with it - and possibly I can look more into it when I will be porting it to newest version of Parsoid.